src/streamlink/plugins/mitele.py

```diff
@@ -54,28 +59,40 @@ def can_handle_url(cls, url):
     def _get_streams(self):
         channel = self._url_re.match(self.url).group("channel")
 
-        pdata = self.session.http.get(self.pdata_url.format(channel=channel),
+        pdata = self.session.http.get(self.caronte_url.format(channel=channel),
                                       acceptable_status=(200, 403, 404),
-                                      schema=self.pdata_schema)
-        log.trace("{0!r}".format(pdata))
+                                      schema=self.caronte_schema)
+        gbx = self.session.http.get(self.gbx_url.format(channel=channel),
+                                    schema=self.gbx_schema)
+
+        log.trace("pdata: {0!r}".format(pdata))
+        log.trace("gbx: {0!r}".format(gbx))
         if pdata.get("code"):
-            log.error("{0} - {1}".format(pdata["code"], pdata["message"]))
+            log.error("error getting pdata: {0}".format(pdata["code"]))
             return
 
-        gdata = self.session.http.post(self.gate_url,
-                                       acceptable_status=(200, 403, 404),
-                                       data=pdata,
-                                       schema=self.gate_schema)
-        log.trace("{0!r}".format(gdata))
-        if gdata.get("code"):
-            log.error("{0} - {1}".format(gdata["code"], gdata["message"]))
+        tokens = self.session.http.post(pdata["cerbero"],
+                                        acceptable_status=(200, 403, 404),
+                                        data=json.dumps({"bbx": pdata["bbx"], "gbx": gbx}),
+                                        headers={
+                                            "content-type": "application/json;charset=UTF-8",
+                                            "origin": "https://www.mitele.es",
+                                            "referer": "https://www.mitele.es"},
+                                        schema=self.cerbero_schema)
+
+        if tokens.get("code"):
+            code = tokens['code']
```

Comment:
Nitpicky, I know :stuck_out_tongue_winking_eye:, but you're mixing quotation styles (down below as well). Also a double attribute lookup.

Reply:
Very. It's cos I refactored an f-string which would have used `'` quotes.